publisher_faq.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Publisher FAQ
+layout: "default"
+---
+
+### If I make a mistake, can I delete a dataset?
+
+You can always go back and edit a dataset but you cannot delete it.  If there is a need to delete a dataset you need to contact your administrator or the Data.gov.uk team (from the contact us link in the site)
```

Comment:
There is now a page about this, so can you append something like "For more information see: [deleting datasets](deleting_datasets.html)